xbmc/GUIInfoManager.cpp

```diff
@@ -1190,7 +1195,8 @@ const infomap system_labels[] =  {{ "hasnetwork",       SYSTEM_ETHERNET_LINK_ACT
                                   { "startupwindow",    SYSTEM_STARTUP_WINDOW },
                                   { "stereoscopicmode", SYSTEM_STEREOSCOPIC_MODE },
                                   { "hasadsp",          SYSTEM_HAS_ADSP },
-                                  { "hascms",           SYSTEM_HAS_CMS }};
+                                  { "hascms",           SYSTEM_HAS_CMS },
+                                  { "haspvraddon",      SYSTEM_HAS_PVR_ADDON }};
```

Comment:
it would make sense if you split those into a separate commit `[gui] new infobool HasPVRAddon`